vite.config.js

```diff
@@ -1,8 +1,15 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 
+const path = require('path')
+
 // https://vitejs.dev/config/
 export default defineConfig({
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src')
```

Comment:
Don't have just a generic alias for src, and src shouldn't be set as an alias
The `@` syntax is good. Let's have:
assets -> `@assets`
components -> `@components`
style -> `@style`
utils -> `@utils`

Reply:
Also, this might not work on storybook. Check if by only changing the vite config the aliases work when used on storybook